Frontend/CO.CDP.OrganisationApp/Pages/Forms/FormsAnswerSetRemoveConfirmation.cshtml.cs

```diff
@@ -24,7 +25,7 @@ public class FormsAnswerSetRemoveConfirmationModel(
     public Guid AnswerSetId { get; set; }
 
     [BindProperty]
-    [Required(ErrorMessage = "Please select an option")]
+    [Required(ErrorMessage = nameof(StaticTextResource.Global_SelectAnOption))]
```

Comment:
I think the format needs to be:     `[Required(ErrorMessageResourceName = nameof(StaticTextResource.ApiKeyManagement_CreateApiKey_ValidationError), ErrorMessageResourceType = typeof(StaticTextResource))]`

Reply:
Ok. Thanks